types/src/block_info.rs

```diff
@@ -1,23 +1,28 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::block::Block;
+use crate::{transaction::Version, validator_set::ValidatorSet};
 use libra_crypto::hash::HashValue;
-use libra_types::{transaction::Version, validator_set::ValidatorSet};
+#[cfg(any(test, feature = "fuzzing"))]
+use libra_crypto::hash::ACCUMULATOR_PLACEHOLDER_HASH;
+#[cfg(any(test, feature = "fuzzing"))]
+use proptest_derive::Arbitrary;
 use serde::{Deserialize, Serialize};
+use std::fmt::{Display, Formatter};
 
 /// The round of a block is a consensus-internal counter, which starts with 0 and increases
-/// monotonically. It is used for the protocol safety and liveness (please see the detailed protocol description).
+/// monotonically.
 pub type Round = u64;
 
 /// This structure contains all the information needed for tracking a block
 /// without having access to the block or its execution output state. It
 /// assumes that the block is the last block executed within the ledger.
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+#[cfg_attr(any(test, feature = "fuzzing"), derive(Arbitrary))]
 pub struct BlockInfo {
     /// Epoch number corresponds to the set of validators that are active for this block.
     epoch: u64,
-    /// The consensus protocol executes proposals (blocks) in rounds, which monotically increase per epoch.
+    /// The consensus protocol is executed in rounds, which monotonically increase per epoch.
```

Comment:
This is okay for now to make the code simpler but in the future, if we go independent signatures on LedgerInfo and QC, we'll want to probably use height for LedgerInfo and also remove any other non-essential fields for LedgerInfo.